frontend/www/js/omegaup/components/user/Dependents.vue

```diff
@@ -25,11 +40,43 @@
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import T from '../../lang';
+import * as ui from '../../ui';
 import { types } from '../../api_types';
 
 @Component
 export default class UserDependents extends Vue {
   @Prop() dependents!: types.UserDependentsPayload[];
+  @Prop() userVerificationDeadline!: Date | null;
+
   T = T;
+  ui = ui;
+
+  get bannerColor() {
+    const today = new Date();
+    const deadline = new Date(this.userVerificationDeadline as Date);
```

Comment:
¿Acá por qué se convierte a `Date` el deadline?

Entiendo que `userVerificationDeadline` puede ser `null`. Me parece que eso indica que el usaurio ya fue verificado, entonces en ese caso debería pintarse en colo verde, no?

Entonces podría quedar algo así
```suggestion
    if (this.userVerificationDeadline  == null) {
      return 'bg-success';
    }
    const deadline = new Date(this.userVerificationDeadline);
```